pipeline/scraper/retriever.js

```diff
@@ -0,0 +1,85 @@
+'use strict';
+//const config = require('/etc/xray/config.json'); //See example_config.json
+const gplay = require('google-play-scraper');
+const logger = require('./logger.js');
+const _ = require('lodash');
+var database = require('./db.js');
+var db = new database('explorer');
+
+const region = 'us';
+
+/**
+ * Inserts app data into the db using db.js
+ * @param {*The app data json that is to be inserted into the databae.} app_data 
+ */
+function insertAppData(app_data) {
+
+    //Checking version data - correct version to update date
+    if (!app_data.version || app_data.version === 'Varies with device') {
+        logger.debug('Version not found defaulting too', app_data.updated);
+        let formatDate = app_data.updated.replace(/\s+/g, '').replace(',', '/');
+        app_data.version = formatDate;
+    }
+
+    // push the app data to the DB
+    return db.insertPlayApp(app_data, region);
+}
+
+function updateSearchedTermDate(searchTerm) {
```

Comment:
this really doesn't need to be a function - that log should probably go into the db module.